Database/schema.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE games(
+    id serial PRIMARY KEY,
+    multiplayer varchar(255),
+    last_palayed_at DATE,
+    publish_date Date,
+    archived Boolean,
```

Comment:
Could you please add additional columns in "game" classes, as per requirements: "As you cannot implement inheritance in the database tables - add all properties and associations from the parent Item class as table columns to all tables based on the child classes. Thanks to that it can be used to store data in the future.".
I mean column: genre, label.

Reply:
Hi @SergSkol PR is already merged and closed. Please suggest.